assets/js/hardworker.js

```diff
@@ -1,77 +1,29 @@
 /* eslint-disable no-undef */
 /* eslint-disable no-unused-vars */
+const removeBtn = document.querySelectorAll('.removeBook');
 const titleInput = document.querySelector('#book_title');
 const authorInput = document.querySelector('#book_author');
 const addBookBtn = document.querySelector('#add_book');
 const addForm = document.querySelector('.addBookForm');
 const bookList = document.querySelector('.list_present');
+let localStorageData = JSON.parse(localStorage.getItem('bookData'));
 
-function addBook() {
-  const titleValue = titleInput.value;
-  const authorValue = authorInput.value;
-
-  if (titleValue !== '' && authorValue !== '') {
-    if (booksData.length === 0) {
-      bookList.innerHTML = '';
-    }
-
-    const bookId = Math.random().toString(36).slice(2);
-    booksData.unshift(
-      {
-        id: bookId,
-        title: titleValue,
-        author: authorValue,
-      },
-    );
-
-    localStorage.setItem('bookData', JSON.stringify(booksData));
-    addForm.reset();
-
-    bookList.innerHTML = `<li>
-                            <h3> ${titleValue} </h3>
-                            <p>by - ${authorValue} </p>
-                            <input type="button" value="Remove" id="${bookId}" class="removeBook" onClick="removeBook('${bookId}')"/>
-                        </li>${bookList.innerHTML}`;
-  }
+if (localStorageData.length === 0 || localStorageData === null) {
```

Comment:
- [ ] In the first instance of the application, I am unable to use your application at all (see image below). This is because of the **`localStorageData.length === 0`** condition. On my own end, on the first running of your app, I do not have `'bookData'` in my `localStorage`, hence that condition will always give me errors. That condition is also somewhat redundant since that condition implies there may be an empty array (the same empty array which you are also setting again in the block). I was able to resolve this by deleting that condition. I would recommend you do the same so that you have:

  ```
    if (localStorageData === null) {
  ```
  ![clint-2](https://user-images.githubusercontent.com/36542986/148036171-1c6da042-fc0c-459e-8f62-8e64a0416bb3.jpg)

Reply:
Thank you for pointing this out, we have resolved this